src/cert_context.rs

```diff
@@ -27,6 +27,16 @@ pub struct CertContext(winapi::PCCERT_CONTEXT);
 unsafe impl Sync for CertContext {}
 unsafe impl Send for CertContext {}
 
+/// A supported hashing algorithm
+#[allow(missing_docs)]
+pub enum HashAlgorithm {
```

Comment:
I don't really like using enums for these kinds of things - it's not extensible to new algorithms as-is and no-one ever really wants to match on it. I like using an opaque wrapper around the underlying `DWORD` or whatever it is.